### PR TITLE
Newtranslation for Bahasa Indonesia

### PR DESCRIPTION
**Pull request by nikofebrianur**
This is my first pull request for Bahasa Indonesia Translation.

### CODE REVIEW

**Comment by ryjones** [comment]
Hi, @nikofebrianur . Please follow these [instructions to fix the DCO issue](https://github.com/hyperledger/fabric-docs-i18n/pull/800/checks?check_run_id=13894458078)

**Review by ryjones — CHANGES_REQUESTED** [review]
Hi, @nikofebrianur . Please follow these [instructions to fix the DCO issue](https://github.com/hyperledger/fabric-docs-i18n/pull/800/checks?check_run_id=13894458078)

File: README.md [hunk 1]
```diff
@@ -1,27 +1,34 @@
-# Hyperledger Fabric Documentation i18n
+# Fabric Docs id_ID
```
Comment (ryjones) [inline README.md]: Please don't change this file; change the one in docs/locale/id_ID

**Comment by ryjones** [comment]
You'll also want to be in docs/locale/id_ID, not id_ID/pt_BR. You might start from 2.5, not 2.2

**Comment by nikofebrianur** [comment]
Hi Ry,

Thanks for your advice, this is my first time to contribute and practice
Git. I am following the docs that say to start from the LTS version. I will
do my best to fix this. Thanks in advance, Ry.
[image: image.png]

**Review by ryjones — APPROVED** [review]

**Comment by ryjones** [comment]
Thank you for contributing!

**Comment by ryjones** [comment]
@nikofebrianur I don't see any content differences - here is the page: https://fabric-docs-id-id.readthedocs.io/id/release-2.5/

**Pull request merged**